app-media-image-capture.html

```diff
@@ -69,9 +69,11 @@
     };
 
     (function() {
+      var supported = true;
```

Comment:
We could probably get some good mileage out of a more descriptive name here, e.g. `imageCaptureSupported`